pypesto/visualize/clust_color.py

```diff
@@ -185,7 +207,7 @@ def assign_colors_for_result_list(num_results, colors=None):
     # if the user did not specify any colors:
     if colors is None:
         dummy_clusters = np.array(list(range(num_results)) * 2)
-        colors = assign_colors(dummy_clusters)
+        colors = assign_colors(dummy_clusters, balance_alpha=False)
```

Comment:
above, we have balance_alpha = True as default?

Reply:
Maybe a clarifying comment might have been nice here, you're implicitly right... ;)
This case should be triggered only if a list of results is plotted. I do not really see the need of balanced alpha here, it might even be unfortunate: The created colors would be used for parameters, waterfalls, profiles, and optimizer history...

Reply:
ok, sounds good.